UPDATING.md

```diff
@@ -52,6 +52,13 @@ assists users migrating to a new version.
 
 ## Master
 
+### Default `[celery] worker_concurrency` is changed to `16`
```

Comment:
Can we also add env variable to set the ''8" value in `entrypoint_ci,sh` - the number of celery workers also impacts the tests run in GitHub so it would be great to keep the lower value there to keep stability.

Reply:
`worker_concurrency` is not the number of workers required -- it is the number of task_instances each Celery Workers will run

Reply:
I have pushed 49a71e12b to change the default for tests to 8 -- but I don't see a reason why it should be less than the default in tests

Reply:
I cannot recall exactly, we can ask @turbaszek, but I believe this was due to stability for some tests in CI that were failing intermittently (likely due to memory issues) when the worker_concurrency was too high and it was easier to change the default value rather than fix the tests (which were simply flaky)